service/internal/cloudconfig/tccpn.go

```diff
@@ -78,11 +79,15 @@ func (t *TCCPN) NewTemplates(ctx context.Context, obj interface{}) ([]string, er
 		} else if err != nil {
 			return nil, microerror.Mask(err)
 		}
+
+		if len(mappings) == 3 {
+			multiMasterEnabled = true
+		}
```

Comment:
maybe there is a better way how to decide if the cluster is with 3 masters?

Reply:
unless you want to check G8SControlPlane, this is how we validate it in OPA

Reply:
We can extend the HA Masters interface later for convenience. All good.